libs/blocks/quiz/utils.js

```diff
@@ -242,11 +278,11 @@ const parseResultData = async (answers) => {
   return rObj;
 };
 
-const getRecomandationResults = (selectedDestination, deafult) =>
-  (selectedDestination.length ? selectedDestination : deafult);
+// eslint-disable-next-line max-len
+const getRecomandationResults = (selectedDestination, deafult) => (selectedDestination.length ? selectedDestination : deafult);
```

Comment:
Not sure whether this grants an ESLint exception, this could be simply split up like
```
const getRecomandationResults = (selectedDestination, deafult) => selectedDestination.length
  ? selectedDestination : default;
```
Also, there's a typo, it should probably be _default_, not _deafult_

Reply:
Thanks, Rares for checking. Fixed the typo and the eslint issue.